src/main/java/edu/harvard/iq/dataverse/FileDownloadServiceBean.java

```diff
@@ -340,6 +328,24 @@ public void explore(GuestbookResponse guestbookResponse, FileMetadata fmd, Exter
         }
     }
 
+    public ApiToken getApiToken(User user, boolean createIfNeeded) {
```

Comment:
Is createIfNeeded ever false? I don't see any usages where it's false

Reply:
Not now. I thought it might be useful somewhere/someday to be able to call this to see if a token exists without triggering creation, but I can remove this if that's over-designing.

Reply:
I just went ahead and removed the second param - easy enough for someone to add in the future.